src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -743,9 +743,13 @@ ProActive User Space mapped using SFTP has the following benefits:
 It has the following constraints and limitations:
 
  * Every user's credentials of the ProActive server must match system users credentials inside the SFTP server. This is typically achieved by enabling LDAP or PAM authentication on the ProActive server (see <<_user_authentication>>).
- * Every file transfer will use the SFTP protocol when it's configured. Note that SFTP protocol will be less efficient than the ProActive protocol when transferring files on the local file system.
+ * Every file transfer will use the SFTP protocol when it's configured. Note that SFTP protocol will be less efficient than the ProActive protocol when transferring files onto and from the local file system.
```

Comment:
I don't know why using the future tense.
Every file transfer use**s** the SFTP protocol when it **is** configured. Note that **the** SFTP protocol **is** less efficient than the ProActive protocol when transferring files onto and from the local file system.

Reply:
Totally agree. Documentation should always use present tense and active sentence forms.

Reply:
Right, I will use present tense